src/helpers.cpp

```diff
@@ -192,7 +192,7 @@ char * Helpers::render_boolean(char * result, bool value) {
 
 // render for native char strings
 char * Helpers::render_value(char * result, const char * value, const int8_t format __attribute__((unused))) {
-    strcpy(result, value);
+    strlcpy(result, value, sizeof(result));
```

Comment:
the sizeof will always be a pointersize (4), not the stringlength.

Reply:
oops, that's bad. I'll fix.